src/multipart.rs

```diff
@@ -317,6 +317,15 @@ where
                 return Ok(Async::NotReady);
             };
 
+            // content disposition
```

Comment:
should we parse content-disposition in `content_disposition` method? I feel, this is too restrictive